osvr/RenderKit/DirectModeVendors.h

```diff
@@ -38,7 +38,10 @@
 #include <algorithm>
 #include <iterator>
 #include <type_traits>
+#include <sstream>
+#include <iomanip>
 #include <utility>
+#include <assert.h>
```

Comment:
I'm pretty sure here there's actually a reason for it to be <assert.h> - it's a weird header (intended for multiple includes, etc) - but I don't remember the reason.